src/components/IframeCssTweaker.js

```diff
@@ -1,12 +1,18 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { web3Connection } from '../actions/web3';
 
-export class IframeCssTweaker extends Component {
```

Comment:
I think our policy is to not have default exports

Reply:
es lint rc of this project is using Airbnb and default export is actually enforced by it and used all across the app

Reply:
I'll check if I can refactor it

Reply:
Yes. Default exports are used everywhere in the App. Refactoring this might require time. I created a ticket for that to handle this when we have time #228 https://github.com/Neufund/ico-transparency-monitor/issues/228